src/test/scala/com/gu/certw/services/CertificateServiceSpec.scala

```diff
@@ -0,0 +1,140 @@
+                 |                        "vendor": "aws",
+                 |                        "credentials": "arn:aws:iam::7784783487:role/PrismAccess-Prism"
+                 |                    }
+                 |                }
+                 |            }
+                 |        ]
+                 |    }
+                 |}""".stripMargin
+
+    val expectedCertificates = List(
+      Certificate(
+        arn = "arn:aws:acm:eu-west-1:7784783487:certificate/blah",
+        domainName = "somedomain.com",
+        status = "EXPIRED",
+        inUseBy = List("arn:aws:elasticloadbalancing:eu-west-1:7784783487:loadbalancer/bleh"),
+        notAfter = None,
+        ownerId = "someOwner",
+        accountName = "someAccount"),
+      Certificate(
+        arn = "arn:aws:acm:eu-west-1:7784783487:certificate/blah2",
+        domainName = "somedomain2.com",
+        status = "ISSUED",
+        inUseBy = List("arn:aws:elasticloadbalancing:eu-west-1:7784783487:loadbalancer/bleh2"),
+        notAfter = Some(ZonedDateTime.parse("2017-10-06T12:00:00.000Z")),
+        ownerId = "someOwner2",
+        accountName = "someAccount2"))
+
+    val service = new CertificateService(httpClient(json), null)
+    val certificates = service.listCertificates
+    certificates shouldEqual expectedCertificates
+  }
+
+  it should "only keep the certificates that are about to expire (29 days)" in {
+    val service = new CertificateService(httpClient(""), null)
+    val c1 = Certificate(
+      arn = "arn:aws:acm:eu-west-1:7784783487:certificate/blah",
+      domainName = "somedomain.com",
+      status = "ISSUED",
+      inUseBy = List("arn:aws:elasticloadbalancing:eu-west-1:7784783487:loadbalancer/bleh"),
+      notAfter = Some(ZonedDateTime.parse("2017-10-06T12:00:00.000Z")),
+      ownerId = "someOwner",
+      accountName = "someAccount")
+    val c2 = c1.copy(notAfter = Some(ZonedDateTime.parse("2017-10-07T12:00:00.000Z")))
+
+    val certificates = List(c1, c2)
+    val expiringSoon = service.soonToExpire(certificates, now = ZonedDateTime.parse("2017-09-08T12:00:00.000Z"))
+    expiringSoon shouldEqual List(c1)
+  }
+
+  it should "ignore certificate when their status is other than ISSUED and EXPIRED" in {
+    val service = new CertificateService(httpClient(""), null)
+    val c1 = Certificate(
+      arn = "arn:aws:acm:eu-west-1:7784783487:certificate/blah",
+      domainName = "somedomain.com",
+      status = "ISSUED",
+      inUseBy = List("arn:aws:elasticloadbalancing:eu-west-1:7784783487:loadbalancer/bleh"),
+      notAfter = Some(ZonedDateTime.parse("2017-10-06T12:00:00.000Z")),
+      ownerId = "someOwner",
+      accountName = "someAccount")
+    val c2 = c1.copy(status = "EXPIRED")
+    val c3 = c1.copy(status = "SOMETHING ELSE")
+
+    val certificates = List(c1, c2, c3)
+    val expiringSoon = service.soonToExpire(certificates, now = ZonedDateTime.parse("2017-09-10T12:00:00.000Z"))
+    expiringSoon shouldEqual List(c1, c2)
```

Comment:
I'd rather see this written as something like "does not contain c3" to make it clearer how this relates to the test message.